### DISCUSSION TITLE
Sudden security inspection during running

### DISCUSSION THREAD

**Discussion by Arctomachine** [FakeFishGames/Barotrauma#14187] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

When security suddenly decide to inspect player who is running past them, it leads to breaking inspection and arrest. Too little time to react and stop.

### Reproduction steps

1. Run past outpost security
2. Trigger security inspection in that moment
3. Not enough time to stop, character keeps running and breaks free from security, fails inspection
4. Security starts arresting character

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.5.8.0 (Summer Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
We've now implemented a fix for this in our private repo, hotfix should be coming up soon!
